### PR TITLE
increase PE # for 35day run

### PR DESCRIPTION
**Pull request by jiandewang**
# PR Checklist

- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [ ] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [ ] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

Provide a detailed description of what this PR does. What bug does it fix, or what feature does it add? Is a change of answers expected from this PR? Are any library updates included in this PR (modulefiles etc.)?

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes #<issue_number>
- fixes noaa-emc/fv3atm/issues/<issue_number>

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [ ] hera.intel
- [ ] hera.gnu
- [ ] orion.intel
- [ ] cheyenne.intel 
- [ ] cheyenne.gnu
- [ ] gaea.intel 
- [ ] jet.intel
- [ ] wcoss_cray
- [ ] wcoss_dell_p3

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on noaa-emc/nems/pull/<pr_number>
- waiting on noaa-emc/fv3atm/pull/<pr_number>

### CODE REVIEW

**Comment by jiandewang** [comment]
@DeniseWorthen I modified the two "35day" templates in order to speed up the runs, otherwise it won't be able to finish the 35-day runs

**Comment by DeniseWorthen** [comment]
@jiandewang This PR needs to be made to the P7a release [branch](https://github.com/ufs-community/ufs-weather-model/tree/release/P7a) 

Also, the number of write tasks per group should be a multiple of [6.](https://github.com/ufs-community/ufs-weather-model/wiki/Advanced-Topics-for-developers#changing-the-number-of-pes-for-fv3) You have 88.

**Comment by jiandewang** [comment]
@DeniseWorthen will merge to P7a. In the meantime I hope you can bring these changes in your upcoming PR so that develop branch will be updated. Adding extra 2 PE in write task won't impact the job, this will let fv3 uses exact 62 nodes.

**Comment by DeniseWorthen** [comment]
Neverless, as I understand it the write tasks are not load balanced when they are not a multiple of ntiles.

Two questions: 
1) what upcoming PR are you referring to ?
2) the dt-atmos for P7a was the same as P6. Why are so many more tasks required? Or does this match the tasks that were actually used in P6 with the workflow?

**Comment by jiandewang** [comment]
I mean https://github.com/ufs-community/ufs-weather-model/pull/584 which I believe is your upcoming PR. Current PE setting requires 20 min/day. The PE changes I made are exactly from P6.

**Comment by DeniseWorthen** [comment]
OK, I understand about the task numbers now. 

We do want this PR to go to the P7a release branch I think. My PR will come until after several others (one is Moorthis so that may take a while). I can add it there too.

**Comment by jiandewang** [comment]
I will get them to P7a branch first, while you can get them to develop branch later

**Comment by DeniseWorthen** [comment]
@jiandewang I've brought the P7a script changes into my CICE PR branch [#610](https://github.com/ufs-community/ufs-weather-model/pull/610). Please make sure they look right.

File: tests/tests/cpld_bmarkfrac_wave_v16_35d [hunk 1]
```diff
@@ -55,6 +55,16 @@ export ocn_petlist_bounds="600 839"
 export ice_petlist_bounds="840 887"
 export wav_petlist_bounds="888 1009"
 
+# 12x16 setting
+export TASKS=1700
+export INPES=12
+export JNPES=16
```
Comment (jiandewang) [inline tests/tests/cpld_bmarkfrac_wave_v16_35d:61]: @DeniseWorthen can you swap INPES and JNPES PE number as 16x12 ?

File: tests/tests/cpld_bmarkfrac_wave_v16_noahmp_35d [hunk 2]
```diff
@@ -46,14 +46,24 @@ export WRTTASK_PER_GROUP=24
...
+#export JNPES=12
+#export med_petlist_bounds="0 575"
+#export atm_petlist_bounds="0 599"
+#export ocn_petlist_bounds="600 839"
+#export ice_petlist_bounds="840 887"
+#export wav_petlist_bounds="888 1009"
+
+# 12x16 setting
+ export TASKS=1700
+ export INPES=12
```
Comment (jiandewang) [inline tests/tests/cpld_bmarkfrac_wave_v16_noahmp_35d:60]: same change needed here

File: tests/tests/cpld_bmarkfrac_wave_v16_35d (hunk 1 above)
Comment (DeniseWorthen) [reply tests/tests/cpld_bmarkfrac_wave_v16_35d:61]: What I have in the _35d tests is identical to ufs-weather mini-release P7a branch. I just made a minor fix to change the indent and comment out the un-used 8x12 settings. 

```
# 12x16 setting
export TASKS=1700
export INPES=16
export JNPES=12
export med_petlist_bounds="0 1151"
export atm_petlist_bounds="0 1239"
export ocn_petlist_bounds="1240 1459"
export ice_petlist_bounds="1460 1539"
export wav_petlist_bounds="1540 1699"
```

Comment (jiandewang) [reply tests/tests/cpld_bmarkfrac_wave_v16_35d:61]: this looks fine

**Comment by DeniseWorthen** [comment]
Changes will be committed in PR #610

**Pull request closed without merging**